packages/hubspot_client/src/hubspot_client.ts

```diff
@@ -769,6 +769,24 @@ export class HubspotClient {
         }
     }
 
+    /**
+     * Updates existing deal with given data
+     *
+     * @param {String|number} hubspotDealId
+     * @param {Object} data
+     */
+    async updateDeal(hubspotDealId: string | number, data: any): Promise<void> {
```

Comment:
if `data` can be any object, we can use `Record<string, any>`, or even better `Record<string, unknown>`, given we are not working with the properties so we dont care about their type
```suggestion
    async updateDeal(hubspotDealId: string | number, data: Record<string, unknown>): Promise<void> {
```

Reply:
Thanks!

Reply:
looks like it broke the tests because the usage expects strings values, so lets change it to `Record<string, string>`